### ISSUE TITLE
Apply basic formatting to default stack name 

### ISSUE THREAD

**Issue by rancher-max** [rancher/rio#644]
In v0.6.0-alpha.2, running `rio up` without specifying name while in `~/Desktop` directory fails with:
```
FATA[0000] Stack.rio.cattle.io "Desktop" is invalid: metadata.name: Invalid value: "Desktop": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*') 
```

Would be nice to apply some formatting to the default name for the user to prevent this error

**Comment by rancher-max** [comment]
Initial milestone set to v1.0 - Backlog. Please triage and update as needed.

**Comment by StrongMonkey** [comment]
fixed in v0.6.0-alpha3. Name are converted to lowercase now.

**Comment by rancher-max** [comment]
Validated the fix in v0.6.0-alpha.3
